### PR TITLE
VZ-7432 psrctl  more CLI functionality

### PR DESCRIPTION
**Pull request by pfmackin**
Add more functionality to the CLI.  Still a work in progress, but run scenarios is working
Add run scenarios functionality
Change iteration to loop for Env vars and metrics.

### CODE REVIEW

File: tools/psr/backend/workers/doc.go [hunk 1]
```diff
@@ -6,7 +6,7 @@ Package workers contains a set of workers that implement PSR use cases.
 
 # Worker
 Each worker does a single task, such as generate logs. The Worker.Work function is called
-repeatedly by the workmanager.Runner, which implements the iteration loop.  If the worker
+repeatedly by the workmanager.Runner, which implements the loop loop.  If the worker
```
Comment (stevshap) [inline tools/psr/backend/workers/doc.go:9]: loop loop

**Review by stevshap — DISMISSED** [review]

File: tools/psr/backend/workmanager/runner.go [hunk 2]
```diff
@@ -45,17 +45,17 @@ func NewRunner(worker spi.Worker, conf config.CommonConfig, log vzlog.Verrazzano
 	r := runner{Worker: worker, runnerMetrics: &runnerMetrics{
 		loopCount: metrics.MetricItem{
 			Name: "loop_count_total",
-			Help: "The total number of loop iterations executed",
+			Help: "The total number of loop loops executed",
```
Comment (mbenmclean) [inline tools/psr/backend/workmanager/runner.go:48]: loop loops

File: tools/psr/backend/workmanager/runner.go [hunk 3]
```diff
@@ -120,15 +120,15 @@ func (r runner) RunWorker(conf config.CommonConfig, log vzlog.VerrazzanoLogger)
 		}
 		log.GetZapLogger().Sync()
 		durationSecondsTotal := time.Now().Unix() - startTimeSecs
-		atomic.StoreInt64(&r.runnerMetrics.workerIterationNanoSeconds.Val, time.Now().UnixNano()-startIteration)
+		atomic.StoreInt64(&r.runnerMetrics.workerLoopNanoSeconds.Val, time.Now().UnixNano()-startLoop)
 		atomic.StoreInt64(&r.runnerMetrics.workerDurationTotalSeconds.Val, durationSecondsTotal)
-		if r.Worker.WantIterationInfoLogged() {
-			log.Infof("Loop Count: %v, Total seconds from start of first work iteration until now: %v", loopCount, durationSecondsTotal)
+		if r.Worker.WantLoopInfoLogged() {
+			log.Infof("Loop Count: %v, Total seconds from start of first work loop until now: %v", loopCount, durationSecondsTotal)
```
Comment (mbenmclean) [inline tools/psr/backend/workmanager/runner.go:126]: first worker loop

File: tools/psr/manifests/scenarios/opensearch/s1/scenario.yaml [hunk 4]
```diff
@@ -1,18 +1,12 @@
 # Copyright (c) 2022, Oracle and/or its affiliates.
 # Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
 
-name: OpenSearch-S1
+name: OpenSearch-s1
```
Comment (mbenmclean) [inline tools/psr/manifests/scenarios/opensearch/s1/scenario.yaml:4]: opensearch should probably be lowercase

File: tools/psr/manifests/scenarios/opensearch/s2/scenario.yaml [hunk 5]
```diff
@@ -1,15 +1,18 @@
 # Copyright (c) 2022, Oracle and/or its affiliates.
 # Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
 
-name: OpenSearch-S2
+name: OpenSearch-s2
```
Comment (mbenmclean) [inline tools/psr/manifests/scenarios/opensearch/s2/scenario.yaml:4]: opensearch should probably be lowercase

File: tools/psr/manifests/scenarios/opensearch/s3/scenario.yaml [hunk 6]
```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2022, Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+name: OpenSearch-s3
```
Comment (mbenmclean) [inline tools/psr/manifests/scenarios/opensearch/s3/scenario.yaml:4]: opensearch should probably be lowercase

File: tools/psr/psrctl/cmd/run/run.go [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+		fmt.Printf("Failed to run scenario %s: %v", scenarioID, err)
+		return err
+	}
+
+	if err != nil {
+		fmt.Printf("%v", err)
+		return err
+	}
+
+	fmt.Println("Example Scenario successfully installed")
```
Comment (mbenmclean) [inline tools/psr/psrctl/cmd/run/run.go:68]: It probably shouldn't always say Example Scenario right? it should probably print the name of the scenario that was passed in.

File: tools/psr/psrctl/cmd/run/run.go [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+	cmd.Example = helpExample
+
+	cmd.PersistentFlags().StringVarP(&scenarioID, flagScenario, flagsScenarioShort, "", flagScenarioHelp)
+
+	return cmd
+}
+
+// runCmdRun - run the "psrctl run" command
+func runCmdRun(cmd *cobra.Command, vzHelper helpers.VZHelper) error {
+	fmt.Println("Starting example scenario...")
```
Comment (mbenmclean) [inline tools/psr/psrctl/cmd/run/run.go:45]: It probably shouldn't always say Example Scenario right? it should probably print the name of the scenario that was passed in.

**Review by stevshap — APPROVED** [review]

**Review by mbenmclean — APPROVED** [review]

**Review by stevshap — APPROVED** [review]

**Pull request merged**